src/L.Control.Locate.scss

```diff
@@ -4,6 +4,10 @@
     font-size: 1.4em;
     color: #444;
     cursor: pointer;
+    width: fit-content !important;
+    min-width: 30px !important;
+    min-height: 30px !important;
```

Comment:
Can you try to not require the `!important` annotation? It usually leads to problems in other places.